code/game/objects/items/weapons/design_disks/excelsior.dm

```diff
@@ -196,3 +196,15 @@
 		/datum/design/autolathe/ammo/shotgun_beanbag,
 		/datum/design/autolathe/ammo/shotgun_pellet
 	)
+
+/obj/item/computer_hardware/hard_drive/portable/design/excelsior/freeheal
```

Comment:
And how does Excelsior get access to biomatter? Can they order it? Does their autolathe even support it?

Reply:
Their autolathe does, and they can recycle clothing into it. or just steal it from neotheo

Reply:
yeah, I wanted to give excels something to utilize their biomatter capabilities early on. I am considering replacing the bandages and ointments with Excel specific healing packs but I need to figure out the spriting for that.